js/index.js

```diff
@@ -1,3 +1,272 @@
+// Function that handles adding event listeners to the chess board
+function initBoardEvListener() {
+    // First column. 'a' column
+    const a1Board = document.getElementById("a1");
+    a1Board.addEventListener("click", function(event) {
+    })
```

Comment:
Each call to `addEventListener()` is missing a semicolon at the end of the statement. Please add a semicolon at the end of each call to `addEventListener()`.